src/main/java/com/hariharanweb/remoteappiummanager/Server.java

```diff
@@ -43,6 +43,14 @@ public static void main(String[] args) throws IOException {
             });
         });
 
+        path("/device/adblog/start", () -> {
```

Comment:
It should be move above inside existing `/device` [path](https://github.com/hariharanwebmail/RemoteAppiumManager/pull/22/files#diff-766c47a020090f03eda1fbd52c17dc0dL35) and create inner path like below,
```
path("/adblog", () -> {
                 get("/start", deviceController. startADBLog, new JsonTransformer());
                 get("/stop", deviceController.stopADBLog, new JsonTransformer());
              });
```